### PR TITLE
Be less eager to recreate the full text search index for datasets

### PR DESCRIPTION
**Pull request by rjmac**
In particular, only do it if its definition has actually changed.

### CODE REVIEW

File: common-pg/src/main/scala/com/socrata/pg/store/SecondarySchemaLoader.scala [hunk 1]
```diff
@@ -48,22 +50,61 @@ class SecondarySchemaLoader[CT, CV](conn: Connection, dsLogger: Logger[CT, CV],
...
+    // it isn't already what it's supposed to be, and "what it's
+    // supposed to be" is more complicated than for individual column
+    // indices (e.g., if a new column is created, this index needs to
+    // be recreated if it already exists).  In order to find out if we
+    // need to do this, we'll use postgresql's comment facility to tag
+    // the index with a hash of the SQL we're using to define the
+    // index.
+    //
+    // Postgresql does give a way to get SQL for the index in the
+    // pg_indexes table, but it's beenñ normalized and it would
```
Comment (saraid) [inline common-pg/src/main/scala/com/socrata/pg/store/SecondarySchemaLoader.scala:63]: `beenñ`

**Review by saraid — APPROVED** [review]
This makes sense to me. Weird, but makes sense.

File: common-pg/src/main/scala/com/socrata/pg/store/SecondarySchemaLoader.scala (hunk 1 above)
Comment (rjmac) [reply common-pg/src/main/scala/com/socrata/pg/store/SecondarySchemaLoader.scala:63]: Isn't that how one conjugatedñ english past-tense verbs?

Comment (saraid) [reply common-pg/src/main/scala/com/socrata/pg/store/SecondarySchemaLoader.scala:63]: No it isn'tñ.

**Comment by chitang** [comment]
:shipit:

**Pull request merged**